src/metadata-builder/ClosureJunctionEntityMetadataBuilder.ts

```diff
@@ -57,7 +57,7 @@ export class ClosureJunctionEntityMetadataBuilder {
                 args: {
                     target: "",
                     mode: "virtual",
-                    propertyName: "descendant_" + primaryColumn.databaseName,
+                    propertyName: "descendant",
```

Comment:
I know it's temporary , it was just for having the basics working so we can use it!
I'll resolve the conflict ASAP.